portfolio/src/main/webapp/index.html

```diff
@@ -23,7 +23,7 @@ <h1>Mariah Crawford's Portfolio</h1>
       <button onclick="addRandomFunFact()">Generate Fun Fact</button>
       <div id="fun-fact-container"></div>
 
-      <button onclick="getDataJson()">Fetch data from Json!</button>
+      <button onclick="getDataJson()"> View Comments</button>
```

Comment:
Was the extra space before "View" intentional?

Reply:
Done